Add Sprint 2 Gantt Chart Planning and Decisions.xlsx to Project-Management folder

shalinikuruguntla opened the pull request:
This update adds the Sprint 2 Gantt Chart Planning and Decisions Excel file under the Project-Management folder.
It includes the detailed sprint tasks and sub-tasks breakdown, categories, resource allocation, durations, dependencies, dependency types for effective tracking and progress evaluation.

HudaKhalil commented:
can you please add another task assigned to me for creating competitive analysis part of prototype enhancements

HudaKhalil reviewed (changes requested):
can you please add another task assigned to me for creating competitive analysis part of prototype enhancements

shalinikuruguntla commented:
> can you please add another task assigned to me for creating competitive analysis part of prototype enhancements

Thanks, Huda! I’ve added the new sub-task (9.3) for Competitive Analysis under Prototype Enhancements and updated the Excel file accordingly. Please review the latest commit when you get a chance.

HudaKhalil commented:
Reviewed and approved to merge
